src/pretix/static/pretixpresale/js/widget/widget.js

```diff
@@ -1408,11 +1417,12 @@ var shared_root_methods = {
                 root.name = data.name;
                 root.frontpage_text = data.frontpage_text;
             } else if (data.events !== undefined) {
-                root.events = data.events;
+                root.events = (root.events || []).concat(data.events);
```

Comment:
Can you test if this behaves correctly in the following edge case?

* All list styles are set to "list", no calendars
* You embed the widget for an **organizer**, not an event
* You click on an event series in that list

This should open a list of dates in that event series. I have a hunch that it instead will append the list to the existing one :(

Reply:
Thanks, I did not think about the organizer-view! I have a solution ready but it expects the organizer view to not be paginated/not have load-more-functionality – which it currently does not have, but might? Currently working solution is:

`root.events = root.has_more_events && "has_more_events" in data && root.events ? root.events.concat(data.events) : data.events;`

But that looks a bit magic to me. So I would prefer setting a append-option when hitting the load-more button. This should be more robust e.g. when later adding load-more to the organizer list-view as well.